### PR TITLE
Removed a duplicate Harvest source for new timesheet entries

### PR DESCRIPTION
**Pull request by justabuzz**
The description of the existing source for new timesheet entries was a bit confusing and I thought it did something else. Therefore, a few days ago I pushed a PR for a new source for new timesheet entries. I realised now that such a source already existed, so I am pushing now a PR to remove my addition and refine the existing source - small description correction to avoid confusion in the future and adding a `ts` to the emitted events. Along the way I added `ts` also to the new invoices source for Harvest.

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4175"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible
- [ ] [Create subscription webhooks sources (vs polling sources) whenever possible](https://pipedream.com/docs/components/guidelines/#webhook-vs-polling-sources)
- [ ] [Always emit a `summary` with events](https://pipedream.com/docs/components/guidelines/#emit-a-summary)
- [ ] [Use a built-in deduping strategy wherever possible](https://pipedream.com/docs/components/guidelines/#deduping)
- [ ] When building webhook sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#webhook-sources)
- [ ] When building polling sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#polling-sources)

File: components/harvest/sources/new-invoice-entry/new-invoice-entry.mjs [hunk 1]
```diff
@@ -39,6 +39,7 @@ export default {
         {
           id: invoice.id,
           summary: `Invoice number: ${invoice.number}`,
+          ts: new Date.parse(invoice.updated_at),
```
[the remark this answers is not in the document]
Comment (justabuzz) [reply components/harvest/sources/new-invoice-entry/new-invoice-entry.mjs]: @jcortes ha! Good catch! :) Just pushed it. Cheers!

**Comment by DilanAthukorala** [comment]
Verification completed, ready for release.

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**